api/CSSKeyframeRule.json

```diff
@@ -28,31 +43,81 @@
             },
             {
               "version_added": "5",
-              "prefix": "moz"
+              "version_removed": "48",
+              "prefix": "Moz"
             }
           ],
           "ie": {
             "version_added": "10"
           },
-          "opera": {
-            "version_added": "12",
-            "prefix": "o"
-          },
-          "opera_android": {
-            "version_added": true
-          },
-          "safari": {
-            "version_added": "4"
-          },
-          "safari_ios": {
-            "version_added": true
-          },
-          "samsunginternet_android": {
-            "version_added": "5.0"
-          },
-          "webview_android": {
-            "version_added": "45"
-          }
+          "opera": [
+            {
+              "version_added": "18"
+            },
+            {
+              "version_added": "15",
+              "version_removed": "18",
+              "prefix": "WebKit"
+            },
+            {
+              "version_added": "12",
```

Comment:
This doesn't seem right, there's no `CSSKeyframeRule` in Opera 12.1 or 12.16 in my testing on BrowserStack.

Reply:
It isn't exposed on its own in that Opera version; one would access it through `CSSKeyframesRule.cssRules`.  I'll add a test into the collector's custom tests!

Reply:
Ah, so just a problem of a missing interface object.